finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,184 @@
+Fetch account transactions
+===
+Use the Finance API to fetch transactions or transactions of certain types from a merchant's liquid account during a specific period.
+
+> **Note**: A merchant's liquid account contains all confirmed transactions that are already paid out or to be paid out to the merchant. If you want to check preliminary transactions that are being confirmed with the issuing banks (buyers' banks), you can fetch them from a merchant's preliminary account.
+
+<!-- Is there any limit for how old transactions can be fetched? -->  
+
+* [Prerequisites](#prerequisites)
+* [Fetch transactions during a specific period](#fetch-transactions-during-a-specific-period)
+* [Fetch transactions of certain types during a specific period](#fetch-transactions-of-certain-types-during-a-specific-period)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up with the required OAuth scope using [Authorization OAuth2 API](../../authorization.adoc). 
+
+## Fetch transactions during a specific period
+When fetching transactions from a merchant's Zettle account during a specific period, set pagination to avoid a big dataset in a response. The transactions should be fetched from the merchant's liquid account.
+
+
+1. Send a request where you set `limit` as the number of transactions to fetch and `offset` as `0`.
+     
+   ```
+   GET /organizations/self/accounts/{accountTypeGroup}/transactions?start={start_time}&end={end_time}&{limit}={limit_value}&offset={offset_value}
+   ```
+
+   Example:
+   
+   The following example fetches the latest three transactions from the merchant's liquid account from 1 January, 2020 to 5 July, 2020. 
+   
+   Request   
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2020-01-01&end=2020-07-05&limit=3&offset=0
+   ```
+   Response         
+   ```json
+       {
+           "data": [
+               {
+                   "timestamp": "2020-07-04T20:16:44.309+0000",
+                   "amount": 381,
+                   "originatorTransactionType": "CARD_PAYMENT_FEE_REFUND",
+                   "originatingTransactionUuid": "30cef6e2-be09-11ea-a8e4-bce028663c34"
+               },
+               {
+                   "timestamp": "2020-07-04T20:16:44.309+0000",
+                   "amount": -20610,
+                   "originatorTransactionType": "CARD_REFUND",
+                   "originatingTransactionUuid": "30cef6e2-be09-11ea-a8e4-bce028663c34"
+               },
+               {
+                   "timestamp": "2020-06-27T23:52:18.327+0000",
+                   "amount": 649,
+                   "originatorTransactionType": "PAYMENT_FEE", // In case of refund
```

Comment:
What does `PAYMENT_FEE` has to do with refund?

Reply:
This would  probably be better like this to show that amount for fee is positive in case of refund
{
                   "timestamp": "2020-06-27T23:52:18.327+0000",
                   "amount": 649, // Positive in case of refund
                   "originatorTransactionType": "PAYMENT_FEE",